content/documentation/pages/5-feature-guides/1-general/1-applicaiton-metadata.md

```diff
@@ -0,0 +1,280 @@
+---
+path: 'feature-guides/general/application-metadata/'
+title: 'Application Metadata'
+description: 'Create and use application properties metadata'
+---
+
+## Application Metadata
```

Comment:
I think this introduction could be improved upon, more along the lines of https://www.baeldung.com/spring-boot-configuration-metadata#configuration-metadata